tests/diffpy/utils/parsers/test_serialization.py

```diff
@@ -7,20 +7,19 @@
 from diffpy.utils.parsers.loaddata import loadData
 from diffpy.utils.parsers.serialization import deserialize_data, serialize_data
 
-tests_dir = os.path.dirname(os.path.abspath(locals().get("__file__", "file.py")))
-
 
 def test_load_multiple(tmp_path, datafile):
     # Load test data
     targetjson = datafile("targetjson.json")
     generatedjson = tmp_path / "generated_serialization.json"
 
-    tlm_list = os.listdir(os.path.join(tests_dir, "testdata", "dbload"))
-    tlm_list.sort()
+    dbload_dir = os.path.dirname(datafile("dbload/e1.gr"))
```

Comment:
then do these two lines using pathlib and pathlib globing?

Reply:
globing returns a tuple[ndarray, dict] to hdata but serialize_data wants a dict

Reply:
Yep, will use pathlib for both parts